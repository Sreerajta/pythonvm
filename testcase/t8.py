def factorial(n):
 fact=1
 if n==0:
  return fact
 else:
  while(n>1):
   fact=fact*n
   n=n-1
  return fact 



print "function to print factorial of given number:"

print "5 factorial is"
print factorial(5) 

print "3 factorial is"
print factorial(3)

print "10 factorial is"
print factorial(10)


print "0 factorial is"
print factorial(0)
