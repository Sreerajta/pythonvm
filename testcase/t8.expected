function to print factorial of given number:
5 factorial is
120
3 factorial is
6
10 factorial is
3628800
0 factorial is
1
